site2/website/sidebars.json

```diff
@@ -38,6 +38,9 @@
       "io-develop",
       "io-cdc"
     ],
+    "Pulsar Schema": [
```

Comment:
I think we should add `Pulsar Schema` before `Pulsar Functions`. Because schema is the component that will be used by functions, io connectors, and presto.